main/src/main/scala/sbt/internal/SysProp.scala

```diff
@@ -69,6 +69,7 @@ object SysProp {
   def traces: Boolean = getOrFalse("sbt.traces")
   def client: Boolean = getOrFalse("sbt.client")
   def ci: Boolean = getOrFalse("sbt.ci")
+  def allowRoot: Boolean = getOrFalse("sbt.root")
```

Comment:
This is a little ambiguous. It could mean the root directory or the root user.

Reply:
`sbt -Dsbt.rootdirectory=true`?

Reply:
Changed to -Dsbt.rootdir in 563bcb9